src/ray/core_worker/transport/direct_actor_task_submitter.cc

```diff
@@ -580,12 +557,19 @@ void CoreWorkerDirectActorTaskSubmitter::HandlePushTaskReply(
       // If the actor is already dead, immediately mark the task object as failed.
       // Otherwise, start the grace period before marking the object as dead.
```

Comment:
Seem this comment is no longer accurate?

Reply:
updated